cfgov/unprocessed/css/organisms/header.less

```diff
@@ -76,11 +76,14 @@
 
     &_content {
 
+        & > .m-global-search {
```

Comment:
I think you mentioned this can be shortened to `> .m-global-search {`, is that right?